option optimize_for = LITE_RUNTIME;

message LegionSkillInfoAnswer
{
 required uint32 result = 1;
 repeated uint32 skill_id = 2;
 repeated uint32 skill_level = 3;
 repeated uint32 skill_limit = 4;
}
